### DISCUSSION TITLE
Win11 VM running at only 1GHz despite many cores provided

### DISCUSSION THREAD

**Discussion by timmorrison73** [utmapp/UTM#4063] [UTM Help (macOS)] [answered]
Hi, I am very new to UTM so not sure if this is a bug or misconfiguration. When I create a Win 11 Arm64 VM, I gave it 7 cores but inside the actual vm it doesn't show any cores and no matter how many cores I bump it up to, it's always at 1 GHz:
<img width="349" alt="Screen Shot 2022-05-25 at 1 25 25 pm" src="https://user-images.githubusercontent.com/86584989/170173867-ad84075f-8b7f-481f-82f8-9a4504432839.png">

My UTM config using Win11 Arm64 virtualization:
<img width="799" alt="Screen Shot 2022-05-25 at 1 28 02 pm" src="https://user-images.githubusercontent.com/86584989/170173961-910df8f5-47a5-4404-9012-10577e54a365.png">

My Macbook Pro 14:
M1 Max
16g ram
8 Cores

Also are there any guides or documentation I can refer to as a new user on how to configure for the best possible Win11 Arm64b VM performance? I am aware of the official guide but it is just listing what each of the categories are.

**Comment by osy** [comment, marked as answer]
That info is not accurate and doesn’t reflect anything on a virtual machine.
